src/onelogin/saml2/auth.py

```diff
@@ -451,3 +453,9 @@ def __build_signature(self, saml_data, relay_state, saml_type, sign_algorithm=On
 
         signature = dsig_ctx.signBinary(str(msg), sign_algorithm_transform)
         return b64encode(signature)
+
+    def get_last_response_xml(self):
+        return self.__last_response_xml
+
+    def get_last_request_xml(self):
+        return self.__last_request_xml
```

Comment:
1. These methods currently return different types. `get_last_request_xml` returns a string as I expected, but `get_last_response_xml` is returning an XML element.
2. Please add docstrings that specify the return type:

```
        :returns: SAML request XML
        :rtype: string
```